xml/System/Enum.xml

```diff
@@ -4277,19 +4280,22 @@ thisInstance And flag = flag
+<xref:System.Enum.TryParse%60%601%28System.String%2CSystem.Boolean%2C%60%600%40%29> is identical to the <xref:System.Enum.Parse%28System.Type%2CSystem.String%2CSystem.Boolean%29> method, except that instead of throwing an exception, it returns `false` if the conversion fails. It eliminates the need for exception handling when parsing the string representation of an enumeration value.
+
+The `value` parameter contains the string representation of an enumeration member's underlying value or named constant, or a list of named constants or underlying values delimited by commas (,). If `value` includes multiple named constants or values, one or more blank spaces can precede or follow each value, name, or comma in `value`. If `value` is a list, `result` reflects the value of the specified names or underlying values combined with a bitwise `OR` operation. If `value` is the string representation of the name of an enumeration value, the comparison of `value` with enumeration names depends on the `ignoreCase` parameter. If `true`, the comparison is case-insensitive; if `false`, it is case-sensitive.
+
+If `value` is a name that does not correspond to a named constant of `TEnum`, the method returns `false`. If `value` is the string representation of an integer that does not represent an underlying value of the `TEnum` enumeration, the method returns an enumeration member whose underlying value is `value` converted to an integral type. If this behavior is undesirable, call the <xref:System.Enum.IsDefined%2A> method to ensure that a particular string representation of an integer is actually a member of `TEnum`.
 
- If `value` is a name that does not correspond to a named constant of `TEnum`, the method returns `false`. If `value` is the string representation of an integer that does not represent an underlying value of the `TEnum` enumeration, the method returns an enumeration member whose underlying value is `value` converted to an integral type. If this behavior is undesirable, call the <xref:System.Enum.IsDefined%2A> method to ensure that a particular string representation of an integer is actually a member of `TEnum`.
+If the parse operation fails, the `result` parameter contains the default value of the underlying type of `TEnum`. The default value is the member in the enumeration with an assigned value of 0, or the first member of the `TEnum` if no values are assigned, or `0` if values are assigned to each member but none of them is 0.
```

Comment:
```suggestion
If the parse operation fails, the `result` parameter contains the default value of 0, which may not be a member of the underlying `TEnum` enumeration in case no member in the enumeration has an assigned value of 0. If no numeric values are assigned to the named constants in `TEnum`, the default is the first member of the `TEnum`.
```

Reply:
This doesn't sound like the way it behaves... Is this related to a new version released?

Reply:
Nope its same, just trying to make the wording better, sounds like it did not improve 😛, which part sounds so different?

Reply:
To clarify, what exactly is the current behavior? 

For example, what would a failed parse operation return if it was trying to parse to this Enum?
![image](https://user-images.githubusercontent.com/18474647/194960981-c05137f9-fb12-4ba5-aa2b-a87f7fb786dc.png)

Reply:
if try to parse an enum member name that does not exist in the enum list the parse would fail and result would have default value of 0.

For your example `testEnum` if you try `Enum.TryParse<testEnum>("test3", out var result);` parse fails and the result will be defaulted to 0 (non-existing value in the enum members); and as you know if try parse `Enum.TryParse<testEnum>("test2", out var result);` parse succeeds and result will be `test2` (2 numerically)

If the `testEnum` were not numbered:
```cs
    public enum testEnum
    {
        test1,
        test2 
    }
```
then by default, the members would be numbered with 0 and 1 therefore `Enum.TryParse<testEnum>("test3", out var result);` parse fails and the result value will be defaulted to `test1` (still 0 numerically)

Reply:
Got it. Given that, I actually think the original text is clear.

Reply:
Well from the original text the the `'result' parameter contains the default value of the underlying type of TEnum'` section does not sound correct because the default value (0) might not be part of the underlying type of `TEnum`. We have updated the same remarks above like this: 
```suggestion
If the parse operation fails, the `result` parameter contains the default value of 0, which may not be a member of the underlying `TEnum` enumeration. If no values are assigned to the named constants in `TEnum`, the default is equal to the first member of the `TEnum`. Otherwise, the default is equal to the member in the enumeration with an assigned value of 0.
```